code/Controllers/CMSMain.php

```diff
@@ -1009,6 +1009,8 @@ public function Breadcrumbs($unlinked = false)
             )));
         }
 
+        $this->extend('updateBreadcrumbs', $items);
```

Comment:
what about CMSPagesController::Breadcrumbs()? It will construct parent breadcrumbs + extensions, then adds the current page to the end. You won't be able to decorate that.

You could use `$this->beforeExtending('updateBreadcrumbs', function () { });` in CMSPagesController.

Reply:
Strange that I haven't seen this happening while testing, but you're right - maybe I should shift this extension point into CMSPagesController instead

Reply:
![image](https://user-images.githubusercontent.com/5170590/36397277-f1d8bd38-1626-11e8-9ed1-4f3a02132328.png)

Is this the breadcrumbs you're talking about from CMSPagesController?

Reply:
it's in the listview when ParentID is set

Reply:
Ok thanks - I've put the CMSPagesController logic into a beforeExtending callback and pushed